module/VuFind/src/VuFind/Db/Table/PagesFactory.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+/**
+ * Factory for pages table.
+ *
+ * PHP version 7
```

Comment:
This comment should be changed to "PHP version 8" to bring it up to date with current requirements. (There are also a couple other outdated version 7 comments currently in this branch, but I think they are pre-existing problems, and the next time you merge dev into this, they should be resolved).

Reply:
This file  is fixed.